download-libs.sh

```diff
@@ -42,6 +48,7 @@ if [ ! -d closure-compiler/.git ]; then
   fi
   git clone --depth 1 https://github.com/google/closure-compiler closure-compiler
```

Comment:
No particular reason, it was more of a quick hack to fix the build at that point (we hoped it will soon be reversed) than a thought out solution. We'll probably move to submodules at some point, but I have other things on my plate now.